base/mmcif_ma-core.dic

```diff
@@ -828,6 +1083,147 @@ save__ma_template_non_poly.details
     _item_type.code               text
      save_
 
+##########################
+## MA_TEMPLATE_BRANCHED ##
+##########################
+
+save_ma_template_branched
+    _category.description
+;              Data items in the MA_TEMPLATE_BRANCHED category record details about
+               the structural templates for branched polymers used in the 
+               homology/comparative modeling.
+;
+    _category.id                  ma_template_branched
+    _category.mandatory_code      no
+    _category_key.name            '_ma_template_branched.ordinal_id'
+    loop_
+    _category_group.id            'inclusive_group'
+                                  'ma_group'
+    loop_
+    _category_examples.detail
+    _category_examples.case
+# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+;
+    Example 1 -
+;
+;
+    loop_
+    _ma_template_branched.ordinal_id
+    _ma_template_branched.template_id
+    _ma_template_branched.name
+    _ma_template_branched.descriptor_type
+    _ma_template_branched.descriptor_value
+    _ma_template_branched.descriptor_software_id
+    _ma_template_branched.type
+    _ma_template_branched.details
+    # ...  abbreviated ...
+;
+# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+   _pdbx_category_context.type         CONTRIBUTED
+   _pdbx_category_context.category_id  ma_template_branched
+     save_
+
+save__ma_template_branched.ordinal_id
+    _item_description.description 
+;    A unique identifier for the category.
+;    
+    _item.name                  '_ma_template_branched.ordinal_id'
+    _item.category_id             ma_template_branched
+    _item.mandatory_code          yes
+    _item_type.code               int
+     save_
+
+save__ma_template_branched.template_id
+    _item_description.description
+;     Identifier for the template.
+      This data item is a pointer to _ma_template_details.template_id
+      in the MA_TEMPLATE_DETAILS caategory. 
+;
+    _item.name                  '_ma_template_branched.template_id'
+    _item.category_id             ma_template_branched
+    _item.mandatory_code          yes
+    _item_type.code               int
+    _item_linked.child_name     '_ma_template_branched.template_id'
+    _item_linked.parent_name    '_ma_template_details.template_id'
+     save_
+
+save__ma_template_branched.name
+    _item_description.description
+;     Name of the branched polymer. 
+;
+    _item.name                  '_ma_template_branched.name'
+    _item.category_id             ma_template_branched
+    _item.mandatory_code          yes
+    _item_type.code               line
+     save_
+
```

Comment:
`_ma_template_non_poly` has `comp_id` instead of a `name`. Shouldn't we use the same here? Would be duplicated with the chem.comp. table but less ambiguous and easier to fill in for developers - they don't need to "know" or come up wit a name, they could just use an already existing component ID.

Reply:
@bienchen `comp_ids` are for monomers and are relevant for non-polymers. For branched polymers, `comp_ids` have to be provided for each monomer in the branched polymer (see https://mmcif.wwpdb.org/dictionaries/mmcif_pdbx_v50.dic/Categories/pdbx_entity_branch_list.html). In case of [3axh](https://www.rcsb.org/structure/3AXH), the disaccharide has only one type of monomer (GLC) and therefore it is simple. But that is not a general case. See https://www.rcsb.org/structure/8K7R for a better example.